product_docs/docs/pem/9/pem_online_help/04_toc_pem_features/01_dashboards/17_str_replication_dashboard.mdx

```diff
@@ -42,9 +42,10 @@ When accessing the `Streaming Replication Analysis` dashboard for the replica no
 
 The `WAL Archive Status` graph displays WAL activity; the vertical key on the left side of the graph indicates the archive count; times are displayed across the bottom of the graph.
 
-The `WAL Segment Lag` graph displays the segment lag for the replica nodes that are associated with the selected server. The vertical key on the left side of the graph indicates the archive count. Each replica node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node.
+The `WAL Segment Lag` graph displays the segment lag for the replica nodes that are associated with the selected server. The vertical key on the left side of the graph indicates the archive count. Each replica node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node. Please note this data will only be displayed against the replica if it is replicating to another replica through cascading replication.
```

Comment:
Copy edits
```suggestion
The `WAL Segment Lag` graph displays the segment lag for the replica nodes that are associated with the selected server. The vertical key on the left side of the graph indicates the archive count. Each replica node is displayed in a different color on the graph. The `Legend` provides a key to the identity (hostname and port) of each graphed slave node. This data is only displayed against the replica if it is replicating to another replica through cascading replication.
```

Reply:
As discussed on Slack I may suggest another form of words once I've understood the behaviour properly

Reply:
resolved as fixed both edits